### ISSUE TITLE
Release 0.15

### ISSUE THREAD

**Issue by endertunc** [testcontainers/testcontainers-rs#497]
Hello!

I would like to use some of the additions such as https://github.com/testcontainers/testcontainers-rs/commit/e23e6d20a4d138b3d597ec33a18d9b8e739979be but it has been more than a year since the last release. 

Is there any plan to release a new version anytime soon? 

As workaround I am currently using following. but I would prefer an official release.
```
testcontainers      = { git = "https://github.com/testcontainers/testcontainers-rs.git", branch = "dev" }
``` 

Thanks in advance!

**Comment by thomaseizinger** [comment]
Yes, maintaining this has been tough, see https://github.com/testcontainers/testcontainers-rs/issues/347.

I'll try to set some time aside for a release.

> There are also 62 unreleased commits in dev branch.

Mind you, most of these are dependency updates. There are only two changes listed in the changelog currently: https://github.com/testcontainers/testcontainers-rs/blob/dev/CHANGELOG.md

**Comment by endertunc** [comment]
Hi @thomaseizinger thanks the clarification, I missed the #347 :)

**Comment by DDtKey** [comment]
I'd expect to merge these PRs before the release:
- https://github.com/testcontainers/testcontainers-rs/pull/521
- https://github.com/testcontainers/testcontainers-rs/pull/431
- https://github.com/testcontainers/testcontainers-rs/pull/523

https://github.com/testcontainers/testcontainers-rs-modules-community also waits for the release to update the `testcontainers`.

**Comment by thomaseizinger** [comment]
> I'd expect to merge these PRs before the release:
> 
>     * [Bump MSRV #521](https://github.com/testcontainers/testcontainers-rs/pull/521)
> 
>     * [Log error when starting a container fails #431](https://github.com/testcontainers/testcontainers-rs/pull/431)
> 
>     * [docs: mention `testcontainers-modules` #523](https://github.com/testcontainers/testcontainers-rs/pull/523)
> 
> 
> [testcontainers/testcontainers-rs-modules-community](https://github.com/testcontainers/testcontainers-rs-modules-community) also waits for the release to update the `testcontainers`.

Thanks for summarising nicely like that. I've just queued the last PR for merging. Happy to cut a release next.

**Comment by thomaseizinger** [comment]
I've opened a release PR here but it is missing some changelog entries: https://github.com/testcontainers/testcontainers-rs/pull/524

Any volunteers to write them? :)

**Comment by DDtKey** [comment]
Closed with #524
Modules are also updated: [release PR](https://github.com/testcontainers/testcontainers-rs-modules-community/pull/46), [crates.io](https://crates.io/crates/testcontainers-modules)

**Comment by thomaseizinger** [comment]
> Closed with #524.

Mind creating a PR that merges `master` back into `dev`? It seems that job has failed: https://github.com/testcontainers/testcontainers-rs/actions/runs/6413983058/job/17413688312?pr=524

**Comment by DDtKey** [comment]
Sure! https://github.com/testcontainers/testcontainers-rs/pull/527
